<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Material Distribution</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Project:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="projects_cmb"/>
        </item>
        <item>
         <widget class="QPushButton" name="newProj_btn">
          <property name="text">
           <string>New</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Sequence:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="sequences_cmb"/>
        </item>
        <item>
         <widget class="QPushButton" name="newSeq_btn">
          <property name="text">
           <string>New</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>General</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>References</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>From Client:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="refClient_le"/>
            </item>
            <item>
             <widget class="QPushButton" name="addRefClient_btn">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="browseRefClient_btn">
              <property name="text">
               <string>Open</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>From Studio:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="refStudio_le"/>
            </item>
            <item>
             <widget class="QPushButton" name="addRefStudio_btn">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="browseRefStudio_btn">
              <property name="text">
               <string>Open</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Offline:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="offline_le"/>
         </item>
         <item>
          <widget class="QPushButton" name="addOffline_btn">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="browseOffline_btn">
           <property name="text">
            <string>Open</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Story:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="story_le"/>
         </item>
         <item>
          <widget class="QPushButton" name="addStory_btn">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="browseStory_btn">
           <property name="text">
            <string>Open</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Feedback:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="feedback_le"/>
         </item>
         <item>
          <widget class="QPushButton" name="addFeedback_btn">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="browseFeedback_btn">
           <property name="text">
            <string>Open</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>WIPS:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="wips_le"/>
         </item>
         <item>
          <widget class="QPushButton" name="addWips_btn">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="browseWips_btn">
           <property name="text">
            <string>Open</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Shot:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="shots_cmb"/>
      </item>
      <item>
       <widget class="QPushButton" name="newShot_btn">
        <property name="text">
         <string>New</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Footage HIGH:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="footHigh_le"/>
        </item>
        <item>
         <widget class="QPushButton" name="addFootHigh_btn">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="browseFootHigh_btn">
          <property name="text">
           <string>Open</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Footage LOW:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="footLow_le"/>
        </item>
        <item>
         <widget class="QPushButton" name="addFootLow_btn">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="browseFootLow_btn">
          <property name="text">
           <string>Open</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Compo ELEMENTS:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="elements_le"/>
        </item>
        <item>
         <widget class="QPushButton" name="addElement_btn">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="browseElement_btn">
          <property name="text">
           <string>Open</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Animation LIPSYNC:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lipSync_le"/>
        </item>
        <item>
         <widget class="QPushButton" name="addLipSync_btn">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="browseLipSync_btn">
          <property name="text">
           <string>Open</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_20">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Asset:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="assets_cmb"/>
      </item>
      <item>
       <widget class="QPushButton" name="newAsset_btn">
        <property name="text">
         <string>New</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_21">
      <item>
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>Art:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="art_le"/>
      </item>
      <item>
       <widget class="QPushButton" name="addArt_btn">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="browseArt_btn">
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_19">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="transfer_btn">
        <property name="text">
         <string>Transfer Data To Folders</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
